presto-docs/src/main/sphinx/release/release-0.235.rst

```diff
@@ -0,0 +1,64 @@
+=============
+Release 0.235
+=============
+
+General Changes
+_______________
+* Fix named queries where its output does not match its column list (:issue:`14333`).
+* Fix error in analysis phase.
+* Fix ZSTD compression issue with zero row file for missing buckets.
+* Improve performance of ``UNNEST``.
+* Improve ``CREATE FUNCTION`` to allow parameter type list and return type to have a length up to 30k characters.
+* Add the ability to pre-check method bytecode size is within limits.
+* Add support for ``SHOW CREATE FUNCTION``.
+* Add optimization for page projection and filter by extract and compute common subexpressions among all projections first. This optimization can be turned off by session property ``optimize_common_sub_expressions``.
+* Add soft memory limit configuration properties. Soft memory limits are default memory limits given to each query that can be overridden using session properties up to the hard limit set by the existing configuration properties. Available soft memory limit configuration properties are ``query.soft-max-memory-per-node``, ``query.soft-max-total-memory-per-node``, ``query.soft-max-total-memory``, and ``query.soft-max-memory``.
+* Add check to disallow invoking SQL functions in SQL function body.
+* Add config property driver.max-page-partitioning-buffer-count, to limit the total number of buffers per optimized repartitioning operator. The default value is 1,000,000.
```

Comment:
is there any documentation about how/why someone would configure this?

Reply:
CC: @yingsu00  any thoughts on that?

Reply:
@tdcmeehan @rschlussel This number should be set large enough so that no exceptions can occur. I will add documentation to it shortly.

Reply:
@tdcmeehan @rschlussel The PR to add documentation for this property is here https://github.com/prestodb/presto/pull/14542
But it seems like https://prestodb.io/docs/current/admin/properties.html only contains most important config properties. I'm not sure if this one is important enough. What do you think?

Reply:
@rschlussel already approved so let's just follow up with the documentation in a separate thread.  I don't think we need to hold up release notes for it.